test/auth.routes.test.js

```diff
@@ -0,0 +1,102 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const passport = require('passport');
+const proxyquire = require('proxyquire');
+const testServer = require('../utils/testServer');
+
+const { userMock } = require('../utils/mocks/users');
+
+const { apiKeysMock, ApiKeysServiceMock, configMock } = require('../utils/mocks/auth');
+
+
+describe('auth - routes', function () {
+  const route = proxyquire('../components/auth/routes', {
+    './apiKeysService': ApiKeysServiceMock,
+    '../../config': configMock,
+  });
+
+  const request = testServer(route);
+
+  this.beforeEach(() => {
+    this.authenticate = sinon.stub(passport, 'authenticate')
+      .callsFake((strategy, callback) => {
+        if (strategy === 'basic') {
+          return function (req, res, next) {
```

Comment:
You are not using res and next, I ran the test without res and next, why are you using this ? 
<img width="381" alt="Screen Shot 2020-05-27 at 15 18 10" src="https://user-images.githubusercontent.com/9372769/83068502-11745d80-a02e-11ea-9230-f369781ce04a.png">

Reply:
@hyfi06 👆